ClntTransMgr/ClntTransMgr.cpp

```diff
@@ -246,6 +246,8 @@ void TClntTransMgr::removeExpired() {
 	    Log(Info) << "Expired address " << ptrAddr->get()->getPlain()
 		      << " from IA " << ptrIA->getIAID()
 		      << " has been removed from addrDB." << LogEnd;
+            Log(Info) << "Client notify script going to be invoked since expiry occurred for address " << ptrAddr->get()->getPlain() << LogEnd;
```

Comment:
Ok, so if there's normal, temporary and prefix expiring at the same time, this will call the script three times. Can you update the code to use `void TIfaceMgr::notifyScripts(const std::string& scriptName, SPtr<TMsg> question, SPtr<TMsg> reply, TNotifyScriptParams& params)` and call it only once?

Reply:
And if there's couple addresses (happens sometimes in DHCPv6), then the script will be called even more time.

Reply:
I have corrected it.